interpreter/binary/decode.ml

```diff
@@ -808,7 +808,7 @@ let module_ s =
   iterate custom_section s;
   let datas = data_section s in
   iterate custom_section s;
-  require (pos s = len s) s (len s) "junk after last section";
+  require (pos s = len s) s (len s) "duplicate section";
```

Comment:
Hm, I don't think this message is adequate. This can be triggered by arbitrary bytes, not just well-formed, repeated sections. Maybe change to `"unexpected content after last section"`?